### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by Abd-Elhakim-Arabet**
implementation of the operations calsses

### CODE REVIEW

File: api/__init__.py [hunk 1]
```diff
@@ -1,3 +1,2 @@
-from flask import Flask
```
Comment (almajd3713) [inline api/__init__.py]: Why remove it? it is required for making the app var

**Review by almajd3713 — CHANGES_REQUESTED** [review]
check my comment please

File: api/__init__.py (hunk 1 above)
Comment (Abd-Elhakim-Arabet) [reply api/__init__.py]: yeah sry accidental

**Review by almajd3713 — APPROVED** [review]
LGTM

**Pull request merged**